Script/test-podspec.sh

```diff
@@ -68,13 +68,13 @@ swift package generate-xcodeproj
 
 # Create a Podfile with our pod as dependency.
 echo "
-platform :tvos, '10.0'
+platform :tvos, '11.0'
 target '$PROJECT_NAME' do
   use_frameworks!
-  pod 'AEPCore', '~> 3.7'
-  pod 'AEPIdentity', '~> 3.7'
-  pod 'AEPServices', '~> 3.7'
-  pod 'AEPRulesEngine', '~> 1.0'
+  pod 'AEPCore', '~> 4'
+  pod 'AEPIdentity', '~> 4'
+  pod 'AEPServices', '~> 4'
+  pod 'AEPRulesEngine', '~> 2'
   pod 'AEPEdgeIdentity', :path => '../AEPEdgeIdentity.podspec'
```

Comment:
Can you clean up the TODO on line 81 by adding `AEPEdge` to the tvOS podspec now that it supports tvOS?

Reply:
here we should also update to ~> 4, let's update all dependencies in the next pr